contracts/Example1.sol

```diff
@@ -1,7 +1,15 @@
 pragma solidity ^0.8.4;
 
 contract Example1 {
-    uint8 public day = 10;
-    uint256 public amount = 1000;
-    address public ownerAddr = 0x4e653b730367e303D30d8FB39d7350D07a60dfD6;
+
+    uint64 public amount;
```

Comment:
uint type of amount should not be changed since the amount can overflow uint64. The expectation was just to reorder